app/models/event.rb

```diff
@@ -0,0 +1,8 @@
+class Event < ApplicationRecord
+  has_one :workshop
```

Comment:
A workshop has a one to many relation to event, please update accordingly. Also, maybe add code to the other model as well (if you need it!)